docs/data/sdks/typescript-browser/migration.md

```diff
@@ -0,0 +1,497 @@
+|`config.deviceId`|`config.deviceId`|
+|`config.deviceIdFromUrlParam`|NOT SUPPORTED|
+|`config.domain`|NOT SUPPORTED|
+|`config.eventUploadPeriodMillis`|`config.flushIntervalMillis`|
+|`config.eventUploadThreshold`|`config.flushQueueSize`|
+|`config.forceHttps`|NOT SUPPORTED|
+|`config.includeFbclid`|NOT SUPPORTED. See [Web attribution](#web-attribution).|
+|`config.includeGclid`|NOT SUPPORTED. See [Web attribution](#web-attribution).|
+|`config.includeReferrer`|NOT SUPPORTED. See [Web attribution](#web-attribution).|
+|`config.includeUtm`|NOT SUPPORTED. See [Web attribution](#web-attribution).|
+|`config.language`|NOT SUPPORTED. See [Plugins](#plugins).|
+|`config.library`|NOT SUPPORTED. See [Plugins](#plugins).|
+|`config.logLevel`|`config.logLevel`|
+|`config.logAttributionCapturedEvent`|NOT SUPPORTED|
+|`config.optOut`|`config.optOut`|
+|`config.onError`|NOT SUPPORTED|
+|`config.onExitPage`|NOT SUPPORTED. See [Flush](#flush-or-onexitpage).|
+|`config.plan`|`config.plan`|
+|`config.plan.branch`|`config.plan.branch`|
+|`config.plan.source`|`config.plan.source`|
+|`config.plan.version`|`config.plan.version`|
+|`config.plan.versionId`|`config.plan.versionId`|
+|`config.platform`|NOT SUPPORTED. See [Plugins](#plugins).|
+|`config.savedMaxCount`|NOT SUPPORTED|
+|`config.saveEvents`|NOT SUPPORTED|
+|`config.saveParamsReferrerOncePerSession`|`config.attribution.trackNewCampaigns`|
+|`config.secureCookie`|`config.cookieSecure`|
+|`config.sessionTimeout`|`config.sessionTimeout`|
+|`config.storage`|`config.storageProvider`|
+|`config.trackingOptions`|`config.trackingOptions`|
+|`config.transport`|`config.transportProvider`|
+|`config.unsetParamsReferrerOnNewSession`|NOT SUPPORTED. Default behavior.|
+|`config.unsentKey`|NOT SUPPORTED|
+|`config.unsentIdentifyKey`|NOT SUPPORTED|
+|`config.uploadBatchSize`|`config.flushQueueSize`|
+|`config.headers`|NOT SUPPORTED|
+|`config.serverZone`|`config.serverZone`|
+|`config.useDynamicConfig`|NOT SUPPORTED|
+|`config.serverZoneBasedApi`|NOT SUPPORTED|
+|`config.sessionId`|`config.sessionId`|
+|`config.partnerId`|`config.partnerId`|
+
+### Tracking events
+
+The APIs to track events the most commonly used. `@amplitude/analytics-browser` offered a variety of `logEvent` APIs like `logEventWithTimestamp`, `logEventWithGroups` to override specific properties in the event payload. Amplitude has simplified all of these variations into a unified `track` API in `@amplitude/analytics-browser`.
+
+#### `logEvent()`
+
+The `logEvent()` API maps to `track()`.
+
+=== "amplitude-js"
+  ```javascript
+  const eventType = "Button Clicked"
+  const eventProperties = {
+    type: "primary",
+  }
+  amplitude.getInstance().logEvent(
+    eventType,
+    eventProperties,
+  )
+  ```
+
+=== "@amplitude/analytics-browser"
+  ```typescript
+  const eventType = "Button Clicked"
+  const eventProperties = {
+    type: "primary",
+  }
+  amplitude.track(
+    eventType,
+    eventProperties,
+  ).promise
```

Comment:
Maybe call out the promise in a separate section. I think people could get confused here and think they need to use the promise.

Reply:
```suggestion
  )
```

Reply:
Removing `.promise` here. There's a dedicated section for callbacks :)